Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2064,15 +2073,22 @@ def lhegensim2018ml(fragment,howMuch):
                           '--era':'Run2_2017',
                           '--conditions':'auto:run2_data'
                           },steps['TIER0']])
-
-steps['TIER0EXPRUN3']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,ALCAPRODUCER:SiPixelCalZeroBias+SiStripCalZeroBias+SiStripCalMinBias+SiStripCalMinBiasAAG+TkAlMinBias,DQM:@express,ENDJOB',
+steps['TIER0EXPRUN3']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,ALCAPRODUCER:@allForExpress+AlCaPCCZeroBiasFromRECO+AlCaPCCRandomFromRECO,DQM:@express,ENDJOB',
```

Comment:
Here I think `ALCAPRODUCER` should be `AlCaPCCRandom`. If I am not mistaken `AlCaPCCZeroBiasFromRECO ` and `AlCaPCCRandomFromRECO ` are not Run-3.

Reply:
I actually wonder if `AlCaPCCRandom` is necessary at all, since it is already included in `@allForExpress`: https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoAlca.py#L54